installers/python-ci/radiasoft-download.sh

```diff
@@ -8,6 +8,7 @@ python_ci_main() {
     fi
     declare i
     declare p=0
+    declare s=1
```

Comment:
Default should be `s=0`.  Only case that needs `s=1` is the default (`*)`).

With two packages, use an array, e.g.

```sh
declare -a p=()
```

Then add to it below:

```sh
p+=( pykern sirepo )
```

Reply:
d86f234e9014fadfaad2084d990b05c3a43068a4